browser/src/control/Control.IdleHandler.ts

```diff
@@ -80,7 +80,7 @@ class IdleHandler {
 			}
 		}
 
-		if (window.mode.isDesktop() && !this.map.uiManager.isAnyDialogOpen()) {
+		if (window.mode.isDesktop() && !this.map.uiManager.isAnyDialogOpen() && $('input:focus').length === 0) {
```

Comment:
we have helper method which is able to detect any input (we have many variants):
https://github.com/CollaboraOnline/online/blob/993af840c00bb329926b14eb27455de9622488bd/browser/src/layer/tile/CanvasTileLayer.js#L3974
I think we need to move it to better place for sharing...